description/dvd.shacl.ttl

```diff
@@ -0,0 +1,146 @@
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix haDVD: <https://data.hetarchief.be/ns/dvd/> .
+@prefix pav: <http://purl.org/pav/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+
+@prefix ebucore: <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
+@prefix haObj: <https://data.hetarchief.be/ns/object/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix schema: <https://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+
+@prefix rel: <http://id.loc.gov/vocabulary/preservation/relationshipSubType/> .
+@prefix premis: <http://www.loc.gov/premis/rdf/v3/> .
+
+<>  a owl:Ontology ;
+    vann:preferredNamespacePrefix "haDVD" ;
+    vann:preferredNamespaceUri "http://data.hetarchief.be/ns/DVD/" ;
+    dct:author  [
+        schema:name   "Miel Vander Sande" ;
+        schema:email  "miel.vandersande@meemoo.be"
+    ] ,
+    [
+        schema:name   "Lennert Van de Velde" ;
+        schema:email  "lennert.vandevelde@meemoo.be"
+    ] ,
+    [
+        schema:name   "Milan Valadou" ;
+        schema:email  "milan.valadou@meemoo.be"
+    ] ;
+    dct:title     "Data model DVD"@en ;
+    dct:title     "Modèle de données DVD"@fr ;
+    dct:title     "Datamodel dvd"@nl ;
+    dct:created   "2024-01-09" ;
+    dct:modified  "2024-01-10" ;
+    dct:description """Data model to provide a detailed description of DVD objects and their components."""@en ;
+    dct:description """Modèle de données pour fournir une description détaillée des objets DVD et leurs composants."""@fr ;
+    dct:description """Datamodel voor de gedetailleerde beschrijving van dvd-objecten en hun componenten."""@nl ;
+    pav:version "0.0.1" .
+
+haDvd:DVDNodeShape a sh:NodeShape ;
+    sh:targetClass haDVD:DVD ;
+    sh:property [
+        a sh:PropertyShape ;
+        sh:path dct:format ;
```

Comment:
Should we better remove it from the other data models as well then? Since it's already in the other descriptive ones, I believe.

Reply:
It's also in the generic descriptive model: https://github.com/viaacode/datamodels/blob/main/description/description.shacl.ttl#L393

Reply:
But shouldn't we remove it from the descriptive model in general as well? Since it's used for internal, ETL-related purposes? We could keep it solely in the specific SHACLs for the ETL that live in the same repo anyway?

Reply:
I removed it from the specific descriptive ontologies and only kept it in the generic descriptive model.